core/src/test/scala/org/apache/spark/status/protobuf/KVStoreProtobufSerializerSuite.scala

```diff
@@ -463,7 +492,7 @@ class KVStoreProtobufSerializerSuite extends SparkFunSuite {
           name = null,
           numPartitions = 8,
           numCachedPartitions = 5,
-          storageLevel = "IN_MEMORY",
+          storageLevel = null,
```

Comment:
Is this change required in this PR?

Reply:
yeah we need to test null string input. There is another object with `storageLevel` as `IN_MEMORY` already. Let's simply change the third object.

Reply:
Got it~